src/components/SearchResultTags.js

```diff
@@ -39,6 +40,10 @@ type SearchResultTagsProps = {
    * Defaults to the value in the configuration.
    */
   baseUri?: string,
+  /** Whether to show a Comment option. Defaults to false. */
+  comments: boolean;
```

Comment:
Still needs to marked as optional here.
```suggestion
  comments?: boolean;
```